scripts/configs.js

```diff
@@ -25,17 +27,30 @@ export const calcEnvironment = () => {
 
 function buildConfigURL(environment) {
   const env = environment || calcEnvironment();
-  const configURL = new URL(`${window.location.origin}/configs.json`);
-  configURL.searchParams.set('sheet', env);
+  let fileName = 'configs.json?sheet=prod';
+  if (env !== 'prod') {
+    fileName = `configs-${env}.json`;
+  }
+  const configURL = new URL(`${window.location.origin}/${fileName}`);
   return configURL;
 }
 
 const getConfigForEnvironment = async (environment) => {
   const env = environment || calcEnvironment();
   let configJSON = window.sessionStorage.getItem(`config:${env}`);
   if (!configJSON) {
-    configJSON = await fetch(buildConfigURL(env)).then((res) => res.text());
-    window.sessionStorage.setItem(`config:${env}`, configJSON);
+    try {
+      configJSON = await fetch(buildConfigURL(env));
+      if (!configJSON.ok) {
+        throw new Error(`Failed to fetch config for ${env}`);
+      }
+      configJSON = await configJSON.text();
+      window.sessionStorage.setItem(`config:${env}`, configJSON);
+    } catch (e) {
+      console.error('Falling back to prod config', e);
+      configJSON = await fetch(buildConfigURL('prod'));
+      configJSON = await configJSON.text();
```

Comment:
I feel like using prod configs when not intended could be problematic - imagine testing order placement locally and accidentally you are posting orders to the prod backend.

Reply:
I get your point :) This fallback is intended for shoppers for when the boilerplate was upgraded without upgrading the content. Alternatively we can also remove the fallback and fail with a meaningful error message. Wdyt?

Reply:
I'm not sure - whatever results in this `catch` being used the least :)

Reply:
Although the idea for a fallback came from me I now agree that it might make more sense to just have an error message. 

For new projects, this case isn't an issue, since it just requires a different setup. In the Alshaya project I saw this code create some confusion, I think the code shouldn't be added to new projects so having it here is confusing to developers.

Reply:
Thanks for the feedback, I removed the fallback.